contracts/libraries/LiquidityPosition.sol

```diff
@@ -317,11 +369,17 @@ library LiquidityPosition {
             .toInt256();
     }
 
+    /// @notice calculates the unrealised funding payment for the position
```

Comment:
Can add dev comment that this function returns vQuoteIncrease (handles the sign for funding payment) - should we update the function name ?